include/deal.II/grid/tria_accessor.h

```diff
@@ -628,6 +628,12 @@ class TriaAccessor : public TriaAccessorBase<structdim, dim, spacedim>
    */
 
   /**
+   * Pointer to the @p ith vertex bounding this object. Throw an exception if
+   * <code>dim=1</code>.
+   */
+  TriaAccessor<0,dim,spacedim> vertex_accessor (const unsigned int i) const;
```

Comment:
Could one change this function to instead return an iterator, like all of the other functions do that allow accessing sub-objects?